python/plugins/processing/algs/gdal/ClipRasterByExtent.py

```diff
@@ -133,6 +139,10 @@ def getConsoleCommands(self, parameters, context, feedback, executing=True):
         arguments.append(str(bbox.xMaximum()))
         arguments.append(str(bbox.yMinimum()))
 
+        crs = inLayer.crs()
+        if override_crs and crs.isValid():
+            arguments.append('-a_srs {}'.format(crs.toProj()))
```

Comment:
This is no safe, better to use `gdal_crs_string()` method here.

Reply:
Thanks for the review. I add the use of `GdalUtils.gdal_crs_string()`.